crates/sui-core/src/authority_server.rs

```diff
@@ -317,7 +317,8 @@ impl ValidatorService {
                 .intent_message
                 .value
                 .kind
-                .input_objects()?
+                .input_objects()
+                .map_err(Into::<SuiError>::into)?
```

Comment:
Does `SuiError::from` not work here?

Reply:
I've not seen this sort of invocation before for `into` and I think `SuiError::from` should just work as there are default trait implementations for anything that implements `Into` for `From` and vice-versa